core/Columns/Updater.php

```diff
@@ -237,7 +236,7 @@ public static function wasDimensionMovedFromCoreToPlugin($name, $version)
             'log_visit.config_realplayer' => 'TINYINT(1) NOT NULL',
             'log_visit.config_device_type' => 'TINYINT( 100 ) NULL DEFAULT NULL',
             'log_visit.visitor_localtime' => 'TIME NOT NULL',
-            'log_visit.location_region' => 'char(2) DEFAULT NULL1',
+            'log_visit.location_region' => 'char(2) DEFAULT NULL1', // TODO: some of these have "NULL1", is this a bug?
```

Comment:
Do you remember why they are set to `NULL1`? Would be useful in a comment in the code.